### ISSUE TITLE
Initially it should be Paused

### ISSUE THREAD

**Issue by ismailkhan89** [pritishvaidya/react-native-flip-timer#5]
i want to make it initially paused then on click button play start the timer.
my code is look like this.
```
`constructor(props){
      super();
      this.state = {
        play: false,
      }
}

play = () => {
      this.setState(({ play }) => ({ play: !play }));
}

render(){
const { play } =this.state
return(
<View>
<Timer time={0} play={play} />
<TouchableOpacity style={styles.button} onPress={this.play}>
          <Text style={styles.text}>{play ? 'Pause' : 'Play'}</Text>
        </TouchableOpacity></View>
)}`
```

**Comment by Abhijeetjaiswalit** [comment]
i don't know why, but its like you need to tigger the state of play.
so if you want it to initially pause, you will have to set it false on componentDidMount.
` componentDidMount(){
        this.setState({
            play:false
        })
    }
`

**Comment by Abhijeetjaiswalit** [comment]
@pritishvaidya setting time={0} on button click, is not setting the timer in initial state(i.e 0:0:0).

**Comment by pritishvaidya** [comment]
It should be fixed in the latest version [0.0.4](https://www.npmjs.com/package/react-native-flip-timer). You just need to update it with the prop `play={false}` as a default. 

The initial state is being set correctly. Please re-check the value. `time={0}`
